src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -1280,6 +1280,40 @@ public static void ContainsNoMatch_StringComparison()
             }
         }
 
+        [Fact]
+        public static void Contains_Match_Char() // Adapted from IndexOf_Match_SingleLetter
+        {
+            Assert.False("".Contains('a'));
+            Assert.False("".AsSpan().Contains('a'));
+
+            // Use a long enough length to incur vectorization code
+            for (var length = 1; length < 250; length++)
+            {
+                char[] a = new char[length];
+                for (int i = 0; i < length; i++)
+                {
+                    a[i] = (char)(i + 1);
+                }
+
+                var str = new string(a);
+                var span = new Span<char>(a);
+                var ros = new ReadOnlySpan<char>(a);
+
+                for (var targetIndex = 0; targetIndex < length; targetIndex++)
+                {
+                    var target = a[targetIndex];
```

Comment:
nit: don't use var here (or at the line below). Only use var where it can easily be inferred from the right-hand side, what the type is.

Reply:
Fixed